content/30.ecosystem/90.node-providers.md

```diff
@@ -53,6 +53,20 @@ access to all providers, as well as other additional features.
 project. With GetBlock you don’t need to know how to run ZKsync nodes as they are already are
 available for mainnet and testnets.
 
+## Infura
+
+ZKsync Era is supported through the
+[DIN](https://www.infura.io/solutions/decentralized-infrastructure-service)
+service, meaning calls to the network are routed to
+[partner infrastructure providers.](https://docs.infura.io/api/networks/zksync#partners-and-privacy-policies)
+
+Infura provides Open Beta access to ZKsync Era. During this period, there might be feature
+limitations. Performance issues are not expected, but they are possible as we optimize
+and stabilize the service.
+[Read more about ZKsync API acccess through Infura.](https://docs.infura.io/api/networks/zksync)
+
+[Sign up for a free Infura account here.](https://docs.infura.io/api)
```

Comment:
maybe this link instead https://docs.infura.io/dashboard? is more clear on how to signup

Reply:
done!